L1Trigger/CSCTriggerPrimitives/python/params/tmbParams.py

```diff
@@ -97,9 +97,17 @@
 
 # to be used by ME21 chambers with GEM-CSC ILT
 tmbPhase2GE21 = tmbPhase2GEM.clone(
-    # matching to GEM clusters with half-strips
+    # match ME1a with GEM (it affects CLCT-GEM match)
+    enableMatchGEMandME1a = cms.bool(True),
+    # match ME1b with GEM (it affects CLCT-GEM match)
+    enableMatchGEMandME1b = cms.bool(True),
+    # matching windows for ALCT-GEM clusters in wiregroups
+    maxDeltaWG = cms.uint32(7),
+    # matching windows for CLCT-GEM clusters in half strip units
     maxDeltaHsEven = cms.uint32(5),
     maxDeltaHsOdd = cms.uint32(10),
+    # efficiency recovery switches
+    dropLowQualityCLCTs_ME1a = cms.bool(True),
     # delay applied in OTMB to GEM trigger primitives
     delayGEMinOTMB = cms.uint32(0)
```

Comment:
When you do a `clone` my understanding is that you should not re-specify the types

Reply:
Actually, what is being cloned doesn't contain those variables yet... I would prefer to keep GE1/1 and GE2/1 parameters separate, since they may change in the future. For now we have only GE1/1 installed and GE2/1 is used for sim studies - thus, prone to change in terms of variables and values.

Reply:
Do you have a `fillDescribtion` method? That would need to contain your new parameters... Maybe for a future PR?

Reply:
I don't think that there's a `fillDescription` method for any of the parameters in those python files (not even the old params related to CSC-only). Shall it be added?